query/Undefiner.java

```diff
@@ -163,7 +162,7 @@ private void undefineSub(ThingType thingType, SubConstraint subConstraint) {
             }
             ThingType supertype = getThingType(subConstraint.type().label().get());
             if (supertype == null) {
-                throw TypeDBException.of(TYPE_NOT_FOUND, subConstraint.type().label().get());
```

Comment:
printing out the `LabelConstraint` leads to an odd printing behaviour, we want to just print `TYPE` does not exist, rather than `TYPE label TYPE` which represents `[variable] label TYPE`